decidim-core/spec/system/messaging/profile_conversations_spec.rb

```diff
@@ -176,12 +176,12 @@
       end
 
       it "shows the topbar button as active" do
-        within "#profile-tabs" do
-          expect(page).to have_selector("li.is-active a", text: "Conversations")
-        end
+        expect(page).to have_selector("li.is-active a", text: "Conversations")
       end
 
       it "shows the topbar button the number of unread messages" do
+        skip "REDESIGN_PENDING - This counter is not implemented. ¿Is the conversations tab inside a group profile deprecated?"
```

Comment:
I just want to highlight this: if it's an open question let's make it to product, I don't think we should leave it in the code

Reply:
We'll tackle this in another issue